docker/app/Dockerfile

```diff
@@ -53,6 +53,7 @@ FROM sillsdev/web-languageforge:base-php AS production-app
 RUN rm /usr/local/bin/install-php-extensions
 RUN apt-get remove -y gnupg2
 RUN mv $PHP_INI_DIR/php.ini-production $PHP_INI_DIR/php.ini
+COPY --from=sillsdev/web-languageforge:wait-latest /wait /wait
```

Comment:
I guess this is an unfortunate requirement to make the e2e tests run off a production image.  I think it's a reasonable compromise, but we should add a comment in the Dockerfile stating why we need "wait" in production.

Reply:
yep, there might be a better way to address this but I can't think of attack vectors this presents so let's go with it.  I'll push a commit with the comment.